crates/solver/src/settlement_submission/submitter/custom_nodes_api.rs

```diff
@@ -17,6 +17,15 @@ use futures::FutureExt;
 use gas_estimation::{EstimatedGasPrice, GasPrice1559};
 use shared::Web3;
 
+const ALREADY_KNOWN_TRANSACTION: [&str; 6] = [
```

Comment:
One thing you can do so you don't need to change the type if you add new messages is:
```suggestion
const ALREADY_KNOWN_TRANSACTION: &[&str] = &[
```

Reply:
Very nice. Thanks.